hivemq/tests/test_e2e.py

```diff
@@ -117,19 +129,108 @@
+    'hivemq.persistence_scheduled_executor.scheduled.percent_of_period.98th_percentile',
+    'hivemq.persistence_scheduled_executor.scheduled.percent_of_period.999th_percentile',
+    'hivemq.persistence_scheduled_executor.scheduled.percent_of_period.99th_percentile',
+    'hivemq.persistence_scheduled_executor.scheduled.percent_of_period.count',
+    'hivemq.persistence_scheduled_executor.scheduled.percent_of_period.max',
+    'hivemq.persistence_scheduled_executor.scheduled.percent_of_period.mean',
+    'hivemq.persistence_scheduled_executor.scheduled.percent_of_period.min',
+    'hivemq.persistence_scheduled_executor.scheduled.percent_of_period.snapshot_size',
+    'hivemq.persistence_scheduled_executor.scheduled.percent_of_period.std_dev',
+    'hivemq.publish.without_matching_subscribers',
+    'hivemq.qos_0_memory.exceeded.per_client',
+    'hivemq.qos_0_memory.max',
+    'hivemq.qos_0_memory.used',
     'hivemq.sessions.overall.current',
     'hivemq.sessions.persistent.active',
     'hivemq.subscriptions.overall.current',
+    'hivemq.system.max_file_descriptor',
+    'hivemq.system.open_file_descriptor',
+    'hivemq.system.os.file_descriptors.max',
+    'hivemq.system.os.file_descriptors.open',
     'hivemq.system.os.global.memory.available',
     'hivemq.system.os.global.memory.swap.total',
     'hivemq.system.os.global.memory.swap.used',
     'hivemq.system.os.global.memory.total',
     'hivemq.system.os.global.uptime',
+    'hivemq.system.os.process.disk.bytes_read',
+    'hivemq.system.os.process.disk.bytes_written',
+    'hivemq.system.os.process.memory.resident_set_size',
     'hivemq.system.os.process.memory.virtual',
     'hivemq.system.os.process.threads.count',
+    'hivemq.system.os.process.time_spent.kernel',
+    'hivemq.system.os.process.time_spent.user',
+    'hivemq.system.physical_memory.free',
+    'hivemq.system.physical_memory.total',
+    'hivemq.system.process_cpu.load',
+    'hivemq.system.process_cpu.time',
+    'hivemq.system.swap_space.free',
+    'hivemq.system.swap_space.total',
+    'hivemq.system.system_cpu.load',
+    'hivemq.topic_alias.count.total',
+    'hivemq.topic_alias.memory.usage',
+    'jvm.buffer_pool.direct.capacity',
+    'jvm.buffer_pool.direct.count',
+    'jvm.buffer_pool.direct.used',
+    'jvm.buffer_pool.mapped.capacity',
+    'jvm.buffer_pool.mapped.count',
+    'jvm.buffer_pool.mapped.used',
+    'jvm.cpu_load.process',
+    'jvm.cpu_load.system',
+    'jvm.gc.cms.count',
+    'jvm.gc.eden_size',
+    'jvm.gc.old_gen_size',
+    'jvm.gc.parnew.time',
+    'jvm.gc.survivor_size',
+    'jvm.heap_memory',
+    'jvm.heap_memory_committed',
+    'jvm.heap_memory_init',
+    'jvm.heap_memory_max',
+    'jvm.loaded_classes',
+    'jvm.non_heap_memory',
+    'jvm.non_heap_memory_committed',
+    'jvm.non_heap_memory_init',
+    'jvm.non_heap_memory_max',
+    'jvm.os.open_file_descriptors',
+    'jvm.thread_count',
```

Comment:
These are duplicated JVM metrics that is extended on L235?

Reply:
Fixed 🙏